tests/org.eclipse.search.tests/plugin.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="org.eclipse.search.textSearchEngine">
+      <textSearchEngine
+            class="org.eclipse.search.tests.TestTextSearchEngine"
```

Comment:
I see an **error** in the IDE now: `Referenced class 'org.eclipse.search.tests.TestTextSearchEngine' in attribute 'class' is not on the plug-in classpath`.

Can we fix that please?

Reply:
This is now fixed with https://github.com/eclipse-platform/eclipse.platform.ui/pull/1137